keyboards/3.0/fr-t-k0-optimise.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<keyboard3 xmlns="https://schemas.unicode.org/cldr/45/keyboard3" locale="fr-t-k0-optimise"
+	conformsTo="45">
+	<!--
+  New French
+-->
+	<version number="1.0.0" />
+	<info author="Team Keyboard" name="French AZERTY optimisé" layout="AZERTY" indicator="FR" />
+
+	<displays>
+		<!-- display for diacritics -->
+		<display output="\u0300" display="${grave}" />
+		<display output="\m{acute}" display="´" />
+		<display output="\m{grave}" display="`" />
+		<display output="\m{umlaut}" display="¨" />
+		<display output="\m{caret}" display="^" />
+		<display output="\m{tilde}" display="~" />
+
+		<!-- display for 'mode' keys -->
+		<display keyId="mark-currency" display="\u{00A4}" />
+		<display keyId="mark-greek" display="μ" />
```

Comment:
The dead key is actually supposed to be the micro (U+00B5) symbol. Double tapping should result in output of the micro symbol; following with m should result in Greek letter mu.